apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V110__add_loaded_batch_loaded_file_idx.sql

```diff
@@ -0,0 +1,8 @@
+/* 
+ BFD-1505
+ There are periodic JDBC errors related to the Loaded Filter Manager bloom filter implementation.
+ 
+ This may correct the issue by having postgres not do a sequential scan on loaded_batches with an index.
+ */
+ 
+${logic.psql-only} create index concurrently if not exists loaded_batches_loaded_file_id_idx on public.loaded_batches using btree (loaded_file_id DESC NULLS LAST);
```

Comment:
probably better to do:

create index ${logic.index-create-concurrently} if not exists loaded_batches_loaded_file_id_idx on loaded_batches (loaded_file_id DESC NULLS LAST);

That way we have the index for both psql and hsql; also btree is the default so no need to specify.